docs/source/apollo-cli.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: The Apollo CLI
```

Comment:
This doc currently lives in the "root" docset, and it's marginally less vestigial in the Rover docs than it is in its current home.